### ISSUE TITLE
Singularity disaster detection email improvements

### ISSUE THREAD

**Issue by stevenschlansker** [HubSpot/Singularity#1292]
Gotta be careful not to make the cure worse than the disease ;)

![screen shot 2016-09-19 at 1 55 12 pm](https://cloud.githubusercontent.com/assets/129097/18648724/be54e6e8-7e70-11e6-9e4e-47cc2ac4e23d.png)

Examples:

```
New disasters detected. Data: SingularityDisastersData{stats=[SingularityDisasterDataPoint{timestamp=1474318365283, numActiveTasks=397, numPendingTasks=18, numLateTasks=1, avgTaskLagMillis=461280, numLostTasks=0, numActiveSlaves=20, numLostSlaves=0}, SingularityDisasterDataPoint{timestamp=1474318355261, numActiveTasks=397, numPendingTasks=18, numLateTasks=1, avgTaskLagMillis=451258, numLostTasks=0, numActiveSlaves=20, numLostSlaves=0}, SingularityDisasterDataPoint{timestamp=1474318345250, numActiveTasks=397, numPendingTasks=18, numLateTasks=1, avgTaskLagMillis=441247, numLostTasks=0, numActiveSlaves=20, numLostSlaves=0}, SingularityDisasterDataPoint{timestamp=1474318335240, numActiveTasks=397, numPendingTasks=18, numLateTasks=1, avgTaskLagMillis=431237, numLostTasks=0, numActiveSlaves=20, numLostSlaves=0}, SingularityDisasterDataPoint{timestamp=1474318325228, numActiveTasks=397, numPendingTasks=18, numLateTasks=1, avgTaskLagMillis=421225, numLostTasks=0, numActiveSlaves=20, numLostSlaves=0}, SingularityDisasterDataPoint{timestamp=1474318315214, numActiveTasks=397, numPendingTasks=18, numLateTasks=1, avgTaskLagMillis=411211, numLostTasks=0, numActiveSlaves=20, numLostSlaves=0}, SingularityDisasterDataPoint{timestamp=1474318305203, numActiveTasks=397, numPendingTasks=18, numLateTasks=1, avgTaskLagMillis=401200, numLostTasks=0, numActiveSlaves=20, numLostSlaves=0}, SingularityDisasterDataPoint{timestamp=1474318295194, numActiveTasks=397, numPendingTasks=18, numLateTasks=1, avgTaskLagMillis=391191, numLostTasks=0, numActiveSlaves=20, numLostSlaves=0}, SingularityDisasterDataPoint{timestamp=1474318285181, numActiveTasks=397, numPendingTasks=18, numLateTasks=1, avgTaskLagMillis=381178, numLostTasks=0, numActiveSlaves=20, numLostSlaves=0}, SingularityDisasterDataPoint{timestamp=1474318275165, numActiveTasks=397, numPendingTasks=18, numLateTasks=1, avgTaskLagMillis=371162, numLostTasks=0, numActiveSlaves=20, numLostSlaves=0}], disasters=[SingularityDisaster{type=EXCESSIVE_TASK_LAG, active=true}, SingularityDisaster{type=LOST_SLAVES, active=false}, SingularityDisaster{type=LOST_TASKS, active=false}, SingularityDisaster{type=USER_INITIATED, active=false}], automatedActionDisabled=false}
```

```
New disasters detected. Data: SingularityDisastersData{stats=[SingularityDisasterDataPoint{timestamp=1474318385305, numActiveTasks=397, numPendingTasks=18, numLateTasks=1, avgTaskLagMillis=481302, numLostTasks=0, numActiveSlaves=20, numLostSlaves=0}, SingularityDisasterDataPoint{timestamp=1474318375295, numActiveTasks=397, numPendingTasks=18, numLateTasks=1, avgTaskLagMillis=471292, numLostTasks=0, numActiveSlaves=20, numLostSlaves=0}, SingularityDisasterDataPoint{timestamp=1474318365283, numActiveTasks=397, numPendingTasks=18, numLateTasks=1, avgTaskLagMillis=461280, numLostTasks=0, numActiveSlaves=20, numLostSlaves=0}, SingularityDisasterDataPoint{timestamp=1474318355261, numActiveTasks=397, numPendingTasks=18, numLateTasks=1, avgTaskLagMillis=451258, numLostTasks=0, numActiveSlaves=20, numLostSlaves=0}, SingularityDisasterDataPoint{timestamp=1474318345250, numActiveTasks=397, numPendingTasks=18, numLateTasks=1, avgTaskLagMillis=441247, numLostTasks=0, numActiveSlaves=20, numLostSlaves=0}, SingularityDisasterDataPoint{timestamp=1474318335240, numActiveTasks=397, numPendingTasks=18, numLateTasks=1, avgTaskLagMillis=431237, numLostTasks=0, numActiveSlaves=20, numLostSlaves=0}, SingularityDisasterDataPoint{timestamp=1474318325228, numActiveTasks=397, numPendingTasks=18, numLateTasks=1, avgTaskLagMillis=421225, numLostTasks=0, numActiveSlaves=20, numLostSlaves=0}, SingularityDisasterDataPoint{timestamp=1474318315214, numActiveTasks=397, numPendingTasks=18, numLateTasks=1, avgTaskLagMillis=411211, numLostTasks=0, numActiveSlaves=20, numLostSlaves=0}, SingularityDisasterDataPoint{timestamp=1474318305203, numActiveTasks=397, numPendingTasks=18, numLateTasks=1, avgTaskLagMillis=401200, numLostTasks=0, numActiveSlaves=20, numLostSlaves=0}, SingularityDisasterDataPoint{timestamp=1474318295194, numActiveTasks=397, numPendingTasks=18, numLateTasks=1, avgTaskLagMillis=391191, numLostTasks=0, numActiveSlaves=20, numLostSlaves=0}], disasters=[SingularityDisaster{type=EXCESSIVE_TASK_LAG, active=true}, SingularityDisaster{type=LOST_SLAVES, active=false}, SingularityDisaster{type=LOST_TASKS, active=false}, SingularityDisaster{type=USER_INITIATED, active=false}], automatedActionDisabled=false}
```

(It'd also be nice to get a formatted email instead of a json dump)

**Comment by ssalinas** [comment]
The de-duping is already implemented in #1278 , will leave open for the email formatting though

**Comment by ssalinas** [comment]
As a note, to disable the emails for the moment, you can use the `emails` section of the smtp configuration and set the list of recipients for `DISASTER_DETECTED` emails to an empty list instead of having it send to admins:
https://github.com/HubSpot/Singularity/blob/master/SingularityService/src/main/java/com/hubspot/singularity/config/SMTPConfiguration.java#L102
